modules/libpng/1.6.41/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - centos7
+  - debian10
+  - debian11
+  - macos
+  - macos_arm64
+  - ubuntu2004
+  - ubuntu2204
+  - windows
+  bazel: [6.x, 7.x]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
```

Comment:
```suggestion
    platform: ${{ platform }}
    bazel: ${{ bazel }}
```

Reply:
Sorry, please remove the duplicated line.

Reply:
done